Parsedown.php

```diff
@@ -1209,7 +1212,7 @@ protected function inlineLink($Excerpt)
 
         if (preg_match('/^[(]((?:[^ ()]|[(][^ )]+[)])+)(?:[ ]+("[^"]*"|\'[^\']*\'))?[)]/', $remainder, $matches))
         {
-            $Element['attributes']['href'] = $matches[1];
+            $Element['attributes']['href'] = htmlspecialchars($matches[1], ENT_QUOTES, 'UTF-8');
```

Comment:
This change conflicts with line 1249 at present (where some less rigorous escaping occurs).

I'd suggest moving line 1249 into the `else` block immediately above it (your changes cause double HTML escaping in some cases). That should solve the test failures you're getting at present.

Alternatively, I guess you could swap the less rigorous escaping on line 1249 with this method (though I'm not sure if that will cause any additional conflicts)